tests-functional/tests/test_get_keycards.py

```diff
@@ -13,7 +13,7 @@ class TestGetKeycards:
     @pytest.fixture(autouse=True)
     def setup_backends(self, backend_new_profile):
         self.account = backend_new_profile("sender")
-        self.keycard = keycard_1
+        self.keycard = copy.deepcopy(keycard_1)
```

Comment:
Please make such fixes in a separate PR next time 🙏 
- smaller PR get merged faster
- independent changes don't hold each other